core/src/main/java/hudson/model/View.java

```diff
@@ -552,18 +549,11 @@ public String getSearchUrl() {
     	List<Action> result = new ArrayList<Action>();
     	result.addAll(getOwner().getViewActions());
     	synchronized (this) {
-    		if (transientActions == null) {
-                updateTransientActions();
-    		}
-    		result.addAll(transientActions);
+            result.addAll(TransientViewActionFactory.createAllFor(this));
     	}
     	return result;
     }
-    
-    public synchronized void updateTransientActions() {
```

Comment:
NB: this is a backwards incompatable API change (however a search of public sources in the jenkinsci org found no usage of it.

Reply:
fixed